ci/ansible/roles/pulp-certs/tasks/main.yml

```diff
@@ -20,13 +20,13 @@
 - name: Configure openssl subjectAltName
   lineinfile: dest=/etc/pki/tls/openssl.cnf
               insertafter="^\[ usr_cert \]"
-              line="subjectAltName=DNS:{{ansible_fqdn}}"
+              line="subjectAltName=DNS:{{ ansible_fqdn }},DNS:{{ ansible_nodename }},DNS:{{ ansible_hostname }}"
```

Comment:
I am not sure the real difference and the source of that information but on a system I was trying I got:

```
        "ansible_nodename": "rhel7-vanilla-np-qeos-88822.slave.openstack.org.novalocal",
        "ansible_fqdn": "rhel7-vanilla-np-qeos-88822.novalocal",
```

Seems that `ansible_fqdn` is `ansible_hostname.ansible_domain` and `ansible_nodename` is what `hostname` command reports.